R/proxy.R

```diff
@@ -37,3 +43,43 @@ compare_proxy.xml_node <- function(x) {
   as.character(x)
 }
 
+# RProtoBuf objects
+#' @export
+compare_proxy.Message <- function(x) {
+  return(x$toString())
```

Comment:
Can you please drop the redundant `return()`?

Reply:
Of course! Sorry, forgot to cleanup the Google-style there.